query-api/src/main/resources/json-schema/projection/range.json

```diff
@@ -30,7 +30,7 @@
                 "type": "integer"
             }
         },
-        "project": {
+        "projection": {
```

Comment:
Accept both project and projection. Keep "project" section, and add "projection" with same contents